rockcraft/commands/init.py

```diff
@@ -16,12 +16,12 @@
 
 """Creation of minimalist rockcraft projects."""
 
+import argparse
```

Comment:
Ruff doesn't like `TYPE_CHECKING` anymore?

Reply:
I'm using it directly in a method signature, so it needs importing.